subnet/etcdv2/local_manager.go

```diff
@@ -33,7 +33,8 @@ const (
 )
 
 type LocalManager struct {
-	registry Registry
+	registry       Registry
+	previousSubnet ip.IP4Net
```

Comment:
What was the reasoning for choosing this mechanism for passing the previousSubnet through to the tryAcquireLease function?